qiita_db/metadata_template/test/test_sample_template.py

```diff
@@ -1862,12 +1864,9 @@ def test_to_dataframe(self):
             'water_content_soil', 'elevation', 'temp', 'tot_nitro',
             'samp_salinity', 'altitude', 'env_biome', 'country', 'ph',
             'anonymized_name', 'tot_org_carb', 'description_duplicate',
-            'env_feature'})
+            'env_feature', 'scientific_name'})
 
     def test_check_restrictions(self):
-        obs = self.tester.check_restrictions([SAMPLE_TEMPLATE_COLUMNS['EBI']])
```

Comment:
Instead of deleting this lines, can you just check that is returning an empty set? I think this was the original idea of this test but got lost when scientific_name become required. This way we will have a test for a template that is not missing anything.